alembic/versions/ee3fa2b93864_refactor_code_structure.py

```diff
@@ -0,0 +1,170 @@
+"""refactor code structure
```

Comment:
Please sync with @bevkololi on the changes that should be included in this file and share here. I ask because these current changes do not seem to reflect those you're introducing in the models.

Reply:
+1

Reply:
![image](https://user-images.githubusercontent.com/12385482/53551122-8dd97180-3b49-11e9-942a-6a390398e183.png)
@RonKbS This is the error I'm getting when I try to drop the tables with `alembic stamp head`. If you have a solution kindly share.

Reply:
I'm also getting kinda the same error. @ThaDeveloper 
![image](https://user-images.githubusercontent.com/38049235/53568828-02c1a100-3b74-11e9-8d74-65ba737c3924.png)
Kindly have a look at this.